CONTRIBUTING.md

```diff
@@ -59,6 +59,36 @@ During the collector release process, all `./.chloggen/*.yaml` files are transcr
 
 Alternately, copy `./.chloggen/TEMPLATE.yaml`, or just create your file from scratch.
 
+## Portable Code
+
+In order to ensure compatibility with different operating systems, code should be portable. Below are some guidelines to follow when writing portable code:
+
+* Avoid using platform-specific libraries, features etc. Please opt for portable multi-platform solutions. 
+
+* Avoid hard-coding platform-specific values. Use environment variables or configuration files for storing platform-specific values.
+
+    For example, avoid using hard-coded file path
+    ```
+    filePath := "C:/Users/Bob/Documents/sampleData.csv"
```

Comment:
```suggestion
    filePath := "C:\Users\Bob\Documents\sampleData.csv"
```

Windows path separator is `\`, unless you were trying to illustrate getting it wrong by assuming all platforms use `/`.

Reply:
Fixed, wasn't intentionally making it wrong, just a typo.